### ISSUE TITLE
Install Error

### ISSUE THREAD

**Issue by PeggyWang0** [sisl/POMDPModelChecking.jl#6]
```
julia> Pkg.add(PackageSpec(url="https://github.com/MaximeBouton/POMDPModelChecking.jl"))
   Cloning git-repo `https://github.com/MaximeBouton/POMDPModelChecking.jl`
  Updating git-repo `https://github.com/MaximeBouton/POMDPModelChecking.jl`
 Resolving package versions...
ERROR: Unsatisfiable requirements detected for package Spot [f11abc24]:
 Spot [f11abc24] log:
 ├─Spot [f11abc24] has no known versions!
 └─restricted to versions * by POMDPModelChecking [abefb91b] — no versions left
   └─POMDPModelChecking [abefb91b] log:
     ├─possible versions are: 0.0.0 or uninstalled
     └─POMDPModelChecking [abefb91b] is fixed to version 0.0.0
Stacktrace:
 [1] check_constraints(::Pkg.GraphType.Graph) at /Users/osx/buildbot/slave/package_osx64/build/usr/share/julia/stdlib/v1.0/Pkg/src/GraphType.jl:935
 [2] Pkg.GraphType.Graph(::Dict{Base.UUID,Set{VersionNumber}}, ::Dict{Base.UUID,Dict{Pkg.Types.VersionRange,Dict{String,Base.UUID}}}, ::Dict{Base.UUID,Dict{Pkg.Types.VersionRange,Dict{String,Pkg.Types.VersionSpec}}}, ::Dict{Base.UUID,String}, ::Dict{Base.UUID,Pkg.Types.VersionSpec}, ::Dict{Base.UUID,Pkg.Types.Fixed}, ::Bool) at /Users/osx/buildbot/slave/package_osx64/build/usr/share/julia/stdlib/v1.0/Pkg/src/GraphType.jl:362
 [3] deps_graph(::Pkg.Types.Context, ::Dict{Base.UUID,String}, ::Dict{Base.UUID,Pkg.Types.VersionSpec}, ::Dict{Base.UUID,Pkg.Types.Fixed}) at /Users/osx/buildbot/slave/package_osx64/build/usr/share/julia/stdlib/v1.0/Pkg/src/Operations.jl:297
 [4] macro expansion at ./logging.jl:310 [inlined]
 [5] resolve_versions!(::Pkg.Types.Context, ::Array{Pkg.Types.PackageSpec,1}) at /Users/osx/buildbot/slave/package_osx64/build/usr/share/julia/stdlib/v1.0/Pkg/src/Operations.jl:339
 [6] #add_or_develop#58(::Array{Base.UUID,1}, ::Function, ::Pkg.Types.Context, ::Array{Pkg.Types.PackageSpec,1}) at /Users/osx/buildbot/slave/package_osx64/build/usr/share/julia/stdlib/v1.0/Pkg/src/Operations.jl:1163
 [7] #add_or_develop at ./none:0 [inlined]
 [8] #add_or_develop#13(::Symbol, ::Bool, ::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::Function, ::Pkg.Types.Context, ::Array{Pkg.Types.PackageSpec,1}) at /Users/osx/buildbot/slave/package_osx64/build/usr/share/julia/stdlib/v1.0/Pkg/src/API.jl:64
 [9] #add_or_develop at ./none:0 [inlined]
 [10] #add_or_develop#12 at /Users/osx/buildbot/slave/package_osx64/build/usr/share/julia/stdlib/v1.0/Pkg/src/API.jl:29 [inlined]
 [11] #add_or_develop at ./none:0 [inlined]
 [12] #add_or_develop#10(::Base.Iterators.Pairs{Symbol,Symbol,Tuple{Symbol},NamedTuple{(:mode,),Tuple{Symbol}}}, ::Function, ::Pkg.Types.PackageSpec) at /Users/osx/buildbot/slave/package_osx64/build/usr/share/julia/stdlib/v1.0/Pkg/src/API.jl:27
 [13] #add_or_develop at ./none:0 [inlined]
 [14] #add#18 at /Users/osx/buildbot/slave/package_osx64/build/usr/share/julia/stdlib/v1.0/Pkg/src/API.jl:69 [inlined]
 [15] add(::Pkg.Types.PackageSpec) at /Users/osx/buildbot/slave/package_osx64/build/usr/share/julia/stdlib/v1.0/Pkg/src/API.jl:69
 [16] top-level scope at none:0
```

**Comment by PeggyWang0** [comment]
```
julia> using POMDPModelChecking
[ Info: Precompiling POMDPModelChecking [abefb91b-a28c-5ab9-9bd9-026e532d7b0e]
[ Info: Precompiling LightGraphs [093fc24a-ae57-5d10-9952-331d41423f4d]
[ Info: Precompiling Spot [f11abc24-ce50-11e8-2475-af6658d13f2b]
ERROR: LoadError: InitError: PyError (PyImport_ImportModule

The Python package spot could not be found by pyimport. Usually this means
that you did not install spot in the Python version being used by PyCall.

PyCall is currently configured to use the Julia-specific Python distribution
installed by the Conda.jl package.  To install the spot module, you can
use `pyimport_conda("spot", PKG)`, where PKG is the Anaconda
package the contains the module spot, or alternatively you can use the
Conda package directly (via `using Conda` followed by `Conda.add` etcetera).

Alternatively, if you want to use a different Python distribution on your
system, such as a system-wide Python (as opposed to the Julia-specific Python),
you can re-configure PyCall with that Python.   As explained in the PyCall
documentation, set ENV["PYTHON"] to the path/name of the python executable
you want to use, run Pkg.build("PyCall"), and re-launch Julia.

) <class 'ModuleNotFoundError'>
ModuleNotFoundError("No module named 'spot'")

Stacktrace:
 [1] pyimport(::String) at /Users/pegasus/.julia/packages/PyCall/0jMpb/src/PyCall.jl:486
 [2] __init__() at /Users/pegasus/.julia/packages/Spot/f5xcR/src/Spot.jl:13
 [3] _include_from_serialized(::String, ::Array{Any,1}) at ./loading.jl:627
 [4] _require_from_serialized(::String) at ./loading.jl:678
 [5] macro expansion at ./logging.jl:317 [inlined]
 [6] _require(::Base.PkgId) at ./loading.jl:941
 [7] require(::Base.PkgId) at ./loading.jl:852
 [8] macro expansion at ./logging.jl:311 [inlined]
 [9] require(::Module, ::Symbol) at ./loading.jl:834
 [10] include at ./boot.jl:317 [inlined]
 [11] include_relative(::Module, ::String) at ./loading.jl:1038
 [12] _require(::Base.PkgId) at ./loading.jl:950
 [13] require(::Base.PkgId) at ./loading.jl:852
 [14] macro expansion at ./logging.jl:311 [inlined]
 [15] require(::Module, ::Symbol) at ./loading.jl:834
during initialization of module Spot
in expression starting at /Users/pegasus/.julia/packages/POMDPModelChecking/NgDPL/src/POMDPModelChecking.jl:14
```

**Comment by MaximeBouton** [comment]
Spot.jl needs to be installed first for this package to work, the README needs to be edited. See https://github.com/sisl/Spot.jl to install Spot.jl.

**Comment by MaximeBouton** [comment]
fix 6f8efb275d8fa3bf046b7b90dc0e225aafb84419
